content/en/docs/troubleshooting/DriftAndDiffDetection.md

```diff
@@ -0,0 +1,113 @@
+---
+title: CRP Drift Detection and Configuration Difference Check Unexpected Result TSG
+description: Troubleshoot situations where CRP drift detection and configuration difference check features are returning unexpected results
+weight: 11
```

Comment:
I wonder why this is not place right (or possibly before) the reportDiff?

Reply:
Oh, let me tweak the rankings